packages/manager/modules/netapp/src/dashboard/snapshot-policies/create/template.html

```diff
@@ -265,114 +333,72 @@
                         </select-checkboxes>
                     </td>
                     <td class="oui-table__cell">
-                        <oui-numeric model="$ctrl.newRule.copies" min="1">
+                        <span
+                            data-ng-bind="::rule.copies"
+                            data-ng-if="!rule.isInEdition"
+                        ></span>
+                        <oui-numeric
+                            data-ng-if="rule.isInEdition"
+                            model="$ctrl.newRule.copies"
+                            min="1"
+                        >
                         </oui-numeric>
                     </td>
                     <td class="oui-table__cell">
                         <oui-button
-                            on-click="$ctrl.validateSnapshotPolicyRule()"
-                            disabled="!$ctrl.isNewRuleValid() || !$ctrl.newSnapshotPolicyForm.$valid || !$ctrl.isRulePrefixAllowed()"
+                            data-ng-if="!rule.isInEdition"
+                            on-click="$ctrl.editRule($index)"
+                            disabled="$ctrl.newRule"
                         >
                             <span
-                                class="oui-icon oui-icon-check"
+                                class="oui-icon oui-icon-pen"
                                 aria-hidden="true"
                             ></span>
                             <span
                                 class="sr-only"
-                                data-translate="netapp_snapshot_policies_create_add_rule_validate"
+                                data-translate="netapp_snapshot_policies_create_rule_delete"
```

Comment:
**question:** `oui-icon-pen` is used for editing, right? Translation key refers to `_rule_delete`. Is it normal?

Reply:
Good catch!